relay-server/src/metric_stats.rs

```diff
@@ -160,22 +161,30 @@ impl MetricStats {
             return None;
         }
 
-        let name = report.metric_name.as_ref()?;
-        let namespace = name.namespace();
-
-        if !namespace.has_metric_stats() {
-            return None;
-        }
```

Comment:
Isn't this important?

Reply:
Yes and no, we can now track cardinality limits which are not associated with a metric (`Type`). The on/off is now only controlled through `report: True` on the cardinality limit, which I think does make sense.

I'll kill the `has_metric_stats` in a follow-up, Jan didn't seem to like it anyways, currently it's still needed for the volume metric 👍 .